stable/mongodb/templates/poddisruptionbudget-arbiter-rs.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.replicaSet.enabled }}
+{{- if and .Values.replicaSet.enabled .Values.replicaSet.pdb.enabled }}
```

Comment:
The parameter `replicaSet.pdb.enabled` doesn't seem to be included in the `README.md`. Could you add it?

Reply:
done!